src/suricata.c

```diff
@@ -237,12 +237,6 @@ int EngineModeIsIPS(void)
     return (g_engine_mode == ENGINE_MODE_IPS);
 }
 
-int EngineModeIsIDS(void)
```

Comment:
would like to keep for completeness of the api

Reply:
ok